pvlib/pvsystem.py

```diff
@@ -2051,7 +2051,7 @@ def snlinverter(v_dc, p_dc, inverter):
     return ac_power
 
 
-def adrinverter(v_dc, p_dc, inverter):
+def adrinverter(v_dc, p_dc, inverter, vtol=0.10):
```

Comment:
vtol : numeric
A unit-less fraction that determines how far the efficiency model is allowed to extrapolate beyond the inverter's normal input voltage operating range.